include/deal.II/base/vector_slice.h

```diff
@@ -156,122 +159,106 @@ make_slice (VectorType &v)
+  : v(v), start(0), length(v.size())
 {}
 
 
 template <typename VectorType>
-inline
-VectorSlice<VectorType>::VectorSlice(VectorType   &v,
-                                     unsigned int start,
-                                     unsigned int length)
-  :
-  v(v), start(start), length(length)
+inline VectorSlice<VectorType>::VectorSlice(VectorType &v,
+                                            unsigned int start,
+                                            unsigned int length)
+  : v(v), start(start), length(length)
 {
-  Assert((start+length<=v.size()),
-         ExcIndexRange(length, 0, v.size()-start+1));
+  Assert((start + length <= v.size()),
+         ExcIndexRange(length, 0, v.size() - start + 1));
 }
 
 
 template <typename VectorType>
-inline
-unsigned int
+inline unsigned int
 VectorSlice<VectorType>::size() const
 {
   return length;
 }
 
 
 template <typename VectorType>
-VectorSlice<VectorType>::
-operator ArrayView<typename VectorType::value_type *> ()
+VectorSlice<VectorType>::operator ArrayView<typename VectorType::value_type *>()
 {
-  return ArrayView<typename VectorType::value_type *> (&v[start], length);
+  return ArrayView<typename VectorType::value_type *>(&v[start], length);
 }
 
 
 template <typename VectorType>
 VectorSlice<VectorType>::
-operator ArrayView<const typename VectorType::value_type *> () const
+operator ArrayView<const typename VectorType::value_type *>() const
 {
-  return ArrayView<const typename VectorType::value_type *> (&v[start], length);
+  return ArrayView<const typename VectorType::value_type *>(&v[start], length);
 }
 
 
 template <typename VectorType>
-inline
-typename VectorType::reference
-VectorSlice<VectorType>::operator[](unsigned int i)
+inline typename VectorType::reference VectorSlice<VectorType>::
+operator[](unsigned int i)
 {
-  Assert ((i<length), ExcIndexRange(i, 0, length));
+  Assert((i < length), ExcIndexRange(i, 0, length));
 
-  return v[start+i];
+  return v[start + i];
 }
 
 
 template <typename VectorType>
-inline
-typename VectorType::const_reference
-VectorSlice<VectorType>::operator[](unsigned int i) const
+inline typename VectorType::const_reference VectorSlice<VectorType>::
+operator[](unsigned int i) const
 {
-  Assert ((i<length), ExcIndexRange(i, 0, length));
+  Assert((i < length), ExcIndexRange(i, 0, length));
 
-  return v[start+i];
+  return v[start + i];
 }
 
 
 template <typename VectorType>
-inline
-typename VectorType::const_iterator
+inline typename VectorType::const_iterator
 VectorSlice<VectorType>::begin() const
 {
-  return v.begin()+start;
+  return v.begin() + start;
 }
 
 
 template <typename VectorType>
-inline
-typename VectorType::iterator
+inline typename VectorType::iterator
 VectorSlice<VectorType>::begin()
 {
-  return v.begin()+start;
+  return v.begin() + start;
 }
 
 
 template <typename VectorType>
-inline
-typename VectorType::const_iterator
+inline typename VectorType::const_iterator
```

Comment:
It doesn't look like there is an option to get `inline` on a separate line. This bothers me more than it should.

Reply:
It should be easy to fix with a little perl script run after clang-indent, though.

Reply:
It seems to keep `inline` on its own line if fixed manually again.